### DISCUSSION TITLE
Welcome to Advance-Ecommerce-in-laravel-7 Discussions!

### DISCUSSION THREAD

**Discussion by Prajwal100** [Prajwal100/Complete-Ecommerce-in-laravel-10#47] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AtulGautam28** [comment]
hello sir your project is really amazing and i am deploying this code on server facing some issues as file manager not found. Because of which I am not getting the image show on the website. Most of all you are requested to tell how these issues will be resolved

**Reply by Thembs0** [reply]
I'm also facing the safe issue.

**Reply by AtulGautam28** [reply]
Then how about another idea.

**Reply by Thembs0** [reply]
Yeah! you can propose one.

**Reply by Prajwal100** [reply]
Hello @AtulGautam28, Really sorry for the late reply. Now i have fixed the image issue. Now you just need to pull the latest code and just click storage link button. 

![image](https://github.com/Prajwal100/Complete-Ecommerce-in-laravel-10/assets/29488275/c1c798c9-37dc-4d82-bf69-971ee26aaeb4)
